### PR TITLE
src/cat/type value for flight UDP

### PR DESCRIPTION
**Pull request by v-roroy**

### CODE REVIEW

File: src/library/simplified/extension/doubleclick-src-type-cat-FIS.js [hunk 1]
```diff
@@ -0,0 +1,435 @@
+if(b.siteId && utag.isFRateDetails() ){
```
Comment (v-roroy) [inline src/library/simplified/extension/doubleclick-src-type-cat-FIS.js:1]: added src, cat, type based on 
https://confluence/display/EWM/Doubleclick+Floodlight+Tag+Configuration

trunk\content\content_config\product\com.expedia.www\flight.ui\tracking.config.xml

File: src/library/simplified/extension/doubleclick-carriercode-flightnumber--u18.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 /**
 b['carrierCodes'] = "SQ|SQ"
 b['flightNumbers'] = "31|402"
-b['u18'] = "SQ31|SQ402"
+b['carrierCodeandFlightNumber'] = "SQ31|SQ402"
```
Comment (v-roroy) [inline src/library/simplified/extension/doubleclick-carriercode-flightnumber--u18.js:4]: UDO variable created as "carrierCodeandFlightNumber" in tealium, hence changing this.

File: src/library/simplified/extension/doubleclick-rewards-u5.js [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+else if(b["isRewardsMember"] != undefined ){
+    b["u5"] = b["isRewardsMember"].toString().charAt(0).toUpperCase() + "|" + "";
+}
+
 
 /** old
  if(utag_data["isRewardsMember"] != undefined && utag_data["rewardsStatus.membershipTier"] != undefined){
   utag_data["u5"] = utag_data["isRewardsMember"].toString().charAt(0) + "|" + utag_data["rewardsStatus.membershipTier"]
 }
+else if(utag_data["isRewardsMember"] != undefined ){
```
Comment (v-roroy) [inline src/library/simplified/extension/doubleclick-rewards-u5.js]: legacy fires u5 as "F|", if the user not logged in.

File: src/library/simplified/extension/doubleclick-src-type-cat-FIS.js [hunk 4]
```diff
@@ -0,0 +1,435 @@
...
+      b['doubleclick.src.fRateDetails'] = "4260726";
+      b['doubleclick.type.fRateDetails'] = "nacat927";
+      b['doubleclick.cat.fRateDetails'] = "nacat262";
+      break;
+   default:
+      break;
+  }
+}
+
+/** old
```
Comment (EXPEanlagnada) [inline src/library/simplified/extension/doubleclick-src-type-cat-FIS.js]: Please delete commented code.

**Pull request merged**